adapters/bmtm/brightmountainmedia_test.go

```diff
@@ -0,0 +1,20 @@
+package bmtm
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderBmtm, config.Adapter{
+		Endpoint: "https://one.elitebidder.com/api/pbs"})
```

Comment:
Nitpick: you can use test url here, not real one. Something like "https://testbmtm.com"

Reply:
Please have a look @VeronikaSolovei9